www/src/components/profile/Profile.js

```diff
@@ -37,8 +37,10 @@ export function Profile() {
   const { dark } = useContext(ThemeContext)
   const [name, setName] = useState(me.name)
   const [email, setEmail] = useState(me.email)
+  const [avatar, setAvatar] = useState(me.avatar)
+  const [avatarFile, setAvatarFile] = useState()
```

Comment:
Using `undefined` here means no changes, `null` will be used to delete it with mutation.